css/index.css

```diff
@@ -143,4 +143,79 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+} 
+
+/*--------------------------------Entire-Page-------------------------------------*/
+
+* {
+    box-sizing: border-box;
+}
+
+.container {
+    margin: 0 auto;
+} 
+
+/*----------------------------------Top-Page--------------------------------------*/
+
+header {
+    display: flex;
+    
+} 
+
+header nav {
```

Comment:
You could use flexbox here instead of the margins on the a tag, but this still works perfectly fine